doc/060_forget.rst

```diff
@@ -336,6 +343,50 @@ could specify:
 (Note that `1w` is not a recognized duration, so you will have to specify 
 `7d` instead)
 
+Security considerations in append-only mode
+===========================================
+
+To prevent a compromised backup client from deleting its backups (for example
+due to a ransomware infection), a repository service/backend can serve the
+repository in a so-called append-only mode. This means that the repository can
+only be written to and read from, while delete and overwrite operations are
+denied. Restic's `rest-server`_ features the append-only mode, but few other
+standard backends do. To support append-only with such a backend, one can use
+`rclone`_ as a complement in between the backup client and the backend service.
+
+.. _rest-server: https://github.com/restic/rest-server/
+.. _rclone: https://rclone.org/commands/rclone_serve_restic/
+
+To remove snapshots and recover the corresponding disk space, the ``forget``
+and ``prune`` commands must have full read, write and delete access to the
+repository. If an attacker has this, the protection offered by append-only
+mode is naturally void.
+
+However, even with append-only mode active, an attacker who is able to add
+additional and empty or otherwise useless snapshots to the repository can
+potentially cause a situation where a trusted client running ``forget`` with
+certain ``--keep-*`` options might unknowingly remove legitimate snapshots,
+leaving only the attackers useless snapshots.
+
+For example, if the ``forget`` policy is to keep three weekly snapshots, and
+the attacker adds an empty snapshot for each of the last three weeks, all with
+a timestamp (see the ``backup`` command's ``-`time`` option) slightly more
+recent than the existing snapshots (but still within the target week), then the
+next time the repository administrator (or scheduled job) runs the ``forget``
```

Comment:
`or a scheduled job`?

Reply:
Yeah, a `forget` run with a policy that is susceptible to the attack discussed here might be run manually by the administrator, or in an automated way by e.g. a cron job or similar. The text said "cron job" but I changed it to "scheduled job" because cron doesn't exist on Windows.

Reply:
Good catch, will change.